db/version_edit_handler.cc

```diff
@@ -10,6 +10,7 @@
 #include "db/version_edit_handler.h"
 
 #include <cinttypes>
+#include <sstream>
```

Comment:
Very minor, but I don't think this is needed.

Reply:
IMO it is better to include this header explicity. At least cppreference mentions that `std::stringstream` is defined in this header:
https://en.cppreference.com/w/cpp/io/basic_stringstream
https://en.cppreference.com/w/cpp/header/sstream
So in the spirit of include-what-you-use I think it is better to include it here too.